score/ipython/scoring_comparison.ipynb

```diff
@@ -0,0 +1,289 @@
+    "# and introducing the risk of misspelling the field name.)\n",
+    "CENSUS_BLOCK_GROUP_ID_FIELD = \"census_block_group_id\"\n",
+    "CENSUS_BLOCK_GROUP_POPULATION_FIELD = \"census_block_group_population\"\n",
+    "CENSUS_TRACT_ID_FIELD = \"census_tract_id\"\n",
+    "CALENVIROSCREEN_SCORE_FIELD = \"calenviroscreen_score\"\n",
+    "CALENVIROSCREEN_PERCENTILE_FIELD = \"calenviroscreen_percentile\"\n",
+    "CALENVIROSCREEN_PRIORITY_COMMUNITY_FIELD = \"calenviroscreen_priority_community\"\n",
+    "\n",
+    "# Note: we are pretending the EJSCREEN's low income percent is the actual score for now as a placeholder.\n",
+    "CEJST_SCORE_FIELD = \"cejst_score\"\n",
+    "CEJST_PERCENTILE_FIELD = \"cejst_percentile\"\n",
+    "CEJST_PRIORITY_COMMUNITY_FIELD = \"cejst_priority_community\"\n",
+    "\n",
+    "# Comparison field names\n",
+    "tract_has_at_least_one_cbg = \"CES Tract has at least one CEJST CBG?\"\n",
+    "tract_has_100_percent_cbg = \"CES Tract has 100% CEJST CBGs?\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2b26dccf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load CEJST score data \n",
+    "cejst_data_path = DATA_DIR / \"score\" / \"csv\" / \"usa.csv\"\n",
+    "\n",
+    "cejst_df = pd.read_csv(cejst_data_path)\n",
+    "\n",
+    "cejst_df.head()\n",
+    "\n",
+    "# Rename unclear name \"id\" to \"census_block_group_id\", as well as other renamings.\n",
+    "cejst_df.rename(columns={\"ID\": CENSUS_BLOCK_GROUP_ID_FIELD,\n",
+    "                         \"ACSTOTPOP\": CENSUS_BLOCK_GROUP_POPULATION_FIELD,\n",
+    "                         \"score_a\": CEJST_SCORE_FIELD,\n",
+    "                         \"score_a_percentile\": CEJST_PERCENTILE_FIELD,\n",
+    "                        }, inplace=True, errors='raise')\n",
+    "\n",
+    "# Calculate the top K% of prioritized communities \n",
+    "cejst_df[CEJST_PRIORITY_COMMUNITY_FIELD] = cejst_df[CEJST_PERCENTILE_FIELD] >= CEJST_PRIORITY_COMMUNITY_THRESHOLD\n",
+    "\n",
+    "# Create the CBG's Census Tract ID by dropping the last number from the FIPS CODE of the CBG. \n",
+    "# The CBG ID is the last one character.\n",
+    "# For more information, see https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html. \n",
+    "cejst_df.loc[:, CENSUS_TRACT_ID_FIELD] = cejst_df.loc[:,CENSUS_BLOCK_GROUP_ID_FIELD].astype(str).str[:-1].astype(np.int64)\n",
+    "\n",
+    "# Remove all non-California data\n",
+    "cejst_df = cejst_df.loc[cejst_df[CENSUS_BLOCK_GROUP_ID_FIELD].astype(str).str[0] == \"6\",:]\n",
+    "\n",
+    "cejst_df.head()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29c14b29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load comparison index (CalEnviroScreen 4)\n",
+    "# TODO: Instead of loading a manually-created CSV from the repo, actually download the data and convert it using \n",
```

Comment:
Cell 3 of the [ejscreen_etl notebook](https://github.com/usds/justice40-tool/blob/main/score/ipython/ejscreen_etl.ipynb) has the download functionality.

Reply:
Good point! I'm gonna leave this for now since I ran into some bot protection services on their site 🤦 : see https://github.com/usds/justice40-tool/issues/194